File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
using namespace std;
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        set<int> s;
        for( int i = 0; i < nums.size(); i++){
            s.insert(nums.at(i));
        }

        if(s.size() == nums.size()){
            return false;
        }
        return true;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        counter = 0
        greatestCounter = counter

        for num in nums:
            if num == 1:
                counter += 1
            else:
                if (counter > greatestCounter):
                    greatestCounter = counter
                counter = 0
        if (counter > greatestCounter):
            greatestCounter = counter
        return greatestCounter
        
